File: lab1/python/ordenar_dos_numeros_2/ordenar_dos_numeros_2.py
num1 = int(input("Introduzca el primer número: "))
num2 = int(input("Introduzca el segundo número: "))

assert num1 > 0
assert num2 > 0

if (num1 > num2):
        num1 = num1 + num2
        num2 = num1 - num2
        num1 = num1 - num2

print(f'{num1} <= {num2}')

File: lab1/c++/ordenar_dos_numeros_2/ordenar_dos_numeros_2.cpp
#include <iostream>

using namespace std;

void ordenar_dos_numeros() {
	int num1, num2;
	
	cout << "Introduzca el primer numero: ";
	cin >> num1;
	
	cout << "Introduzca el segundo numero: ";
	cin >> num2;
	
	if (num1 > 0 && num2 > 0) {
		if (num1 > num2) {
	        num1 = num1 + num2;
	        num2 = num1 - num2;
	        num1 = num1 - num2;
    	}
    	
    	cout << num1 << " <= " << num2; 
	}
	
}

int main() {

	ordenar_dos_numeros();

	return 0;

}
